installation/scripts/testing.sh

```diff
@@ -56,15 +76,28 @@ do
         testExitCode=1
         break
     fi
+
     sec=$((currTime-startTime))
     min=$((sec/60))
     if (( min > 60 )); then
         echo "Timeout for test suite '${suiteName}' occurred."
         testExitCode=1
         break
     fi
-    echo "ClusterTestSuite not finished. Waiting..."
-    sleep 60
+    if (( $minCache != $min )); then
```

Comment:
I am afraid that this code becomes too complicated. Maybe you can focus right now on improving script by more frequent checks if a suite is finished?

Reply:
discussed